circleguard/loadable.py

```diff
@@ -722,3 +722,8 @@ def __str__(self):
             return f"Loaded ReplayPath by {self.username} on {self.map_id} at {self.path}"
         else:
             return f"Unloaded ReplayPath at {self.path}"
+
+    def positionally_translate_replay(self):
```

Comment:
Would've preferred to put this in the test file, but unfortunately, loading of replay data is lazy, so it only loads once you start an analysis. It also resulted in me putting in a bit of a hack to enable positional translation + noise when a certain flag is marked.

Reply:
you can force replay data loading with `cg.load(replay)`, in case you didn't know

Reply:
Sounds good, where do you prefer `positionally_translate_replay` to live?

Reply:
and also, would you want it parameterized? A lot of what i through together was kinda a hack.

Reply:
This is just for testing right? Should probably live in the tests file then, like you suggested